packages/cart-sdk/types.ts

```diff
@@ -18,6 +18,7 @@ export interface Cart {
 
 export interface CartLineItem {
    itemcode: string;
+   variant?: string;
```

Comment:
I suggest using `variantTitle`, since `variant` can be used to refer to the variant object

```suggestion
   variantTitle?: string;
```